forms/aws/ecs.tf

```diff
@@ -40,6 +40,7 @@ data "template_file" "form_viewer_task" {
     google_client_secret     = aws_secretsmanager_secret_version.google_client_secret.arn
     database_url             = aws_secretsmanager_secret_version.database_url.arn
     redis_url                = aws_elasticache_replication_group.redis.primary_endpoint_address
+    token_secret             = awa_secretsmanager_secret_version.token_secret.arn
```

Comment:
You'll also need to add an associated block to [forms/aws/task-definitions/form_viewer.json](https://github.com/cds-snc/forms-staging-terraform/blob/main/forms/aws/task-definitions/form_viewer.json#L70-L73) to make the secret available in the ECS task.